### PR TITLE
Remove cheers and auto_excerpt

### PR DESCRIPTION
**Pull request by stevenjoezhang**
`auto_excerpt` is not a feature that the theme should be responsible for. Please search for and install third-party plug-ins yourself.

### CODE REVIEW

**Comment by ivan-nginx** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ scripts/filters/post.js  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/hexo-theme-next/hexo-theme-next/pullRequest?prid=4553469&bid=15316455)

File: _config.yml [hunk 1]
```diff
@@ -315,9 +309,6 @@ post_navigation: left
 # See: https://theme-next.org/docs/theme-settings/custom-pages
 # ---------------------------------------------------------------
 
-# Enable "cheers" for archive page.
```
Comment (liolok) [inline _config.yml]: Why remove?

Comment (stevenjoezhang) [reply _config.yml]: Yes. Always enabled.

Comment (liolok) [reply _config.yml]: Then it would be very nice (user friendly) to write a simple guide in this PR to tell users **how**.

Comment (stevenjoezhang) [reply _config.yml]: **how**: https://theme-next.org/docs/advanced-settings.html#Injects

**Pull request merged**